packages/combobox/src/ComboboxMenu/Menu.styles.ts

```diff
@@ -8,12 +8,17 @@ import { fontFamilies, spacing, typeScales } from '@leafygreen-ui/tokens';
 import { ComboboxSize as Size } from '../Combobox.types';
 
 export const menuItemPadding: Record<Size, { x: number; y: number }> = {
+  [Size.XSmall]: { x: 12, y: 8 },
+  [Size.Small]: { x: 12, y: 8 },
   [Size.Default]: { x: 12, y: 8 },
   [Size.Large]: { x: 12, y: 8 },
 };
 
 /** Height of a menu item (in px) */
-export const menuItemHeight = {
+export const menuItemHeight: Record<Size, number> = {
+  [Size.XSmall]:
+    typeScales.body1.lineHeight + 2 * menuItemPadding[Size.XSmall].y,
+  [Size.Small]: typeScales.body1.lineHeight + 2 * menuItemPadding[Size.Small].y,
   [Size.Default]:
     typeScales.body1.lineHeight + 2 * menuItemPadding[Size.Default].y,
   [Size.Large]: typeScales.body2.lineHeight + 2 * menuItemPadding[Size.Large].y,
```

Comment:
This object could also be a function if the typesScales.body1 reference was set in a const